app/models/event.rb

```diff
@@ -27,10 +28,14 @@ def location
   end
 
   def title_with_date
-    date = starts_at.strftime("%B %d")
+    return nil if starts_at.blank?
```

Comment:
Added this check because when an event is created in the admin with a empty `starts_at` field administrate calls this method before validation, which produces a no method for nil class error.